jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -29,17 +29,17 @@ def _add_to_table(app, data, comp_label):
         The label used to describe the data. Also used as the column header.
     """
     # Add data to the mos viz table object
-    if 'MOS Table' not in app.data_collection:
+    if "MOS Table" not in app.data_collection:
         table_data = Data(label="MOS Table")
         app.data_collection.append(table_data)
 
-        mos_table = app.data_collection['MOS Table']
+        mos_table = app.data_collection["MOS Table"]
```

Comment:
I believe your IDE undid the previous changes that Nick mentioned (" for text and ' for things that behave like identifiers).